dts/arm/nxp/nxp_ke1xf.dtsi

```diff
@@ -115,6 +117,30 @@
 			status = "disabled";
 		};
 
+		i2c0: i2c@40066000 {
+			compatible = "nxp,kinetis-lpi2c";
```

Comment:
@galak should we reuse the existing compatible `nxp,imx-lpi2c` instead?

Reply:
I think that would be confusing at best. The recommended format for the DTS `compatible` string is `manufacturer,model`, which would not be correct if we used `nxp,imx-lpi2c`.

I suggest we either use `nxp,kinetis-lpi2c` as currently in this patch series - or that we go with the SoC series agnostic `nxp,lpi2c`.

We could also go with a combination of `compatible = "nxp,kinetis-lpi2c", "nxp,lpi2c"`.

Reply:
@galak has generally advised to reuse compatible strings if they have already been defined elsewhere, including Linux, for the same hardware ip. We already have precedent in the i.mx rt series using kinetis compatibles:
https://github.com/zephyrproject-rtos/zephyr/blob/ba4eae14ddea910777e11b1ac139fc147dc192c6/dts/arm/nxp/nxp_rt.dtsi#L243-L250

Unfortunately we don't yet support multiple compatibles in the yaml bindings.

Reply:
Yeah, in the short term we should use the existing compatible.

Reply:
Eventually we need to move to dt-schema which I think will solve the multiple compatible issue.

Reply:
@MaureenHelm @galak Thank you for your input on this. I have changed the commit to use the existing binding for now.